qt/applications/workbench/workbench/plotting/figurewindow.py

```diff
@@ -27,12 +27,14 @@ class FigureWindow(QMainWindow, ObservingView):
     activated = Signal()
     closing = Signal()
     visibility_changed = Signal()
+    show_context_menu = Signal()
     close_signal = Signal()
 
     def __init__(self, canvas, parent=None):
         QMainWindow.__init__(self, parent=parent)
         # attributes
         self._canvas = weakref.proxy(canvas)
+        canvas.installEventFilter(self)
```

Comment:
We shouldn't need this any more I don't think?

Reply:
We likely do not!